src/services/__tests__/country-service.test.ts

```diff
@@ -56,3 +57,30 @@ describe('deleteCountryService', () => {
     expect(Country.findByIdAndDelete).toHaveBeenCalledTimes(1);
   });
 });
+
+// Test update a country
+describe('updateCountryService', () => {
+  it('should update and return the updated country', async () => {
+    const mockCountry = { _id: '123', name: 'naija' };
+
+    // mock the find and update country method to return mock countries
+    (Country.findByIdAndUpdate as jest.Mock).mockResolvedValue(mockCountry);
+
+    //call the service
+    const result = await updateCountryService('123', 'naija');
```

Comment:
Use the mockCountry object you created for the function parameters to follow DRY principle. Call mockCountry._id or mockCountry.name

Reply:
Okay boss